onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -1066,12 +1106,12 @@ class GpuDFTOperatorFactory : public WRL::Base<IMLOperatorKernelFactory>
         }
     }
 
-    static void RegisterDFTKernel(IMLOperatorRegistry* registry)
+    static void RegisterDFTKernel(IMLOperatorRegistry* registry, int32_t version)
     {
         MLOperatorKernelDescription kernelDescription = {};
         kernelDescription.domain = "";
         kernelDescription.name = "DFT";
-        kernelDescription.minimumOperatorSetVersion = 17;
+        kernelDescription.minimumOperatorSetVersion = version;
```

Comment:
curious - why did the "minimum" changes? isn't the min still 17?

Reply:
The minimum doesn't change, minimumOperatorSetVersion is the only field that governs which version is being used (there is no operatorSetVersion field).